### PR TITLE
[#IOPID-280] Save audit logs

### PR DESCRIPTION
**Pull request by BurnedMarshal**
#### List of Changes

Add blob storage service
Save blob document on success fast login request

#### Motivation and Context

We need to build an audit log for fast login valid requests, a documents containing all the required information will be saved on a Blob Storage.

#### How Has This Been Tested?

unit test

#### Types of changes

- [X] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,13 +51,16 @@
   "dependencies": {
     "@azure/cosmos": "^3.17.3",
     "@azure/functions": "^3.5.1",
+    "@azure/storage-blob": "^12.15.0",
```
Comment (arcogabbo) [inline package.json]: I think we can safely remove this
```suggestion
```

File: src/functions/__tests__/fast-login.test.ts [hunk 2]
```diff
@@ -220,10 +248,55 @@ describe("Fast Login handler", () => {
...
+        }
+      };
+      const result = await makeFastLoginHandler({
+        ...httpHandlerInputMocks,
+        input: req,
+        fnLollipopClient: mockedFnLollipopClient,
+        blobService: mockBlobService
+      })();
+      expect(getAssertionMock).toBeCalled();
+      expect(mockUpsertBlobFromObject).toBeCalled();
```
Comment (arcogabbo) [inline src/functions/__tests__/fast-login.test.ts:306]: I think we can enrich this test including the arguments passed to the `upsertBlobFromObject` function
```suggestion
      expect(mockUpsertBlobFromObject).toBeCalled();
      expect(mockUpsertBlobFromObject).toBeCalledWith(
        {},
        "logs",
        expect.stringContaining(
          validLollipopHeaders["x-pagopa-lollipop-user-id"]
        ),
        expect.objectContaining({
          assertion_xml: aSAMLResponse,
          client_ip: validFastLoginAdditionalHeaders["x-pagopa-lv-client-ip"]
        })
      );
```

**Review by arcogabbo — APPROVED** [review]
LGTM. just minor suggestions

File: src/functions/__tests__/fast-login.test.ts (hunk 2 above)
Comment (BurnedMarshal) [reply src/functions/__tests__/fast-login.test.ts:306]: Fixed in [fe0d4b2](https://github.com/pagopa/io-functions-fast-login/pull/15/commits/fe0d4b260714df1876e67003dfc749d7410a4373)

File: package.json (hunk 1 above)
Comment (BurnedMarshal) [reply package.json]: Fixed in [fe0d4b2](https://github.com/pagopa/io-functions-fast-login/pull/15/commits/fe0d4b260714df1876e67003dfc749d7410a4373)

**Pull request merged**